app/controllers/candidate_interface/work_history_controller.rb

```diff
@@ -0,0 +1,24 @@
+module CandidateInterface
+  class WorkHistoryController < CandidateInterfaceController
+    def length
+      @work_details_form = WorkHistoryForm.new
+    end
+
+    def submit_length
+      @work_details_form = WorkHistoryForm.new(work_history_params)
+
+      @work_details_form.valid?
+      render :length
+    end
+
+  private
+
+    def work_history_params
+      return nil unless params.has_key?(:candidate_interface_work_history_form)
```

Comment:
For a form that only has a radio button (and no text fields for example), submitting it won't even include the `candidate_interface_work_history_form[]` hash as part of the payload, so the require below will blow up.